tutorials/breakoutboard-tut.md

```diff
@@ -0,0 +1,90 @@
+
+### Digital outputs
+
+![Breakout board digital output processing graph](../images/breakout-tut-6.png)
+
+The 8-bit digital output port is updated using a <xref:OpenEphys.Onix1.BreakoutDigitalOutput> operator. A ~10 Hz timer is used to drive a counter. Although this produced a 32-bit integer that counts from 1 to 2147483647,  <xref:OpenEphys.Onix1.BreakoutDigitalOutput> operator will only use the lower 8-bits to update the digital output state. This can be seen reflected on the LEDs on the breakout board will show a binary count for 0 to 255 before the pattern repeats.
+
+### Memory monitor
+
+![Breakout board memory monitor processing graph](../images/breakout-tut-7.png)
+
+The hardware first-in-first-out (FIFO) memory use is monitored using 8-bit digital output port is updated using a <xref:OpenEphys.Onix1.MemoryMonitorData> operator. A snapshot of the hardware FIFO's use is taken at regular intervals at a rate determined in the configuration [breakout board configuration](#configurebreakoutboard). The <xref:OpenEphys.Onix1.MemoryMonitorDataFrame>s are saved and re-emitted by a produced by a `CsvWriter`. They can then be expanded by right-click the operator and hovering over `Output` in the context menu, in this case to access the `PercentUsed`, which shows the amount of the percent of hardware FIFO that occupied by data. This is a diagnostic data stream that is most useful in the context of closed-loop performance. It tells the user if data is being consumed rapidly enough by the host PC to keep up with data production by the hardware. The hardware FIFO is a buffer that is required to deal with the fact that computers with normal operating systems cannot perform operations with strict regularity. When there are hiccups in acquisition, the hardware FIFO picks up the slack, but should then be cleared immediately. To get the lowest latencies, the `BlockReadSize` should be as small as possible *while the memory use percentage remains around 0%*.
+
+> [!WARNING]
+> If the hardware FIFO's `PercentUsed` is non-zero for long time periods, or is increasing, the `BlockReadSize` is setting is too small (see [StartAcquisition](#startacquisition)). A small `BlockReadSize` will mean that the host computer does not have to wait long for enough data to become available to propagate it forward, but will reduce overall bandwidt by increasing the frquency at which the host computer checks if data is available and performs hardware reads.
```

Comment:
"the `BlockReadSize` **is** setting is too small" => "the `BlockReadSize` setting is too small"

"but will reduce overall **bandwidt** by increasing the **frquency**" => "but will reduce overall bandwidth by increasing the frequency"